packages/slider/src/SliderHandle.ts

```diff
@@ -131,6 +135,16 @@ export class SliderHandle extends Focusable {
         super.update(changes);
     }
 
+    protected firstUpdated(changedProperties: PropertyValues<this>): void {
+        super.firstUpdated(changedProperties);
+        const { max, min } = this as { max: number; min: number };
+        if (!this.value) {
```

Comment:
We should specifically check for missing value, since `!0` is `true`:

```suggestion
        if (this.value == null) {
```